apps/docs/src/config.ts

```diff
@@ -57,6 +57,7 @@ export const SIDEBAR: Sidebar = {
       { text: 'Testimonials', link: 'testimonials' },
       { text: 'Avatars', link: 'avatars' },
       { text: 'Typography', link: 'typography' },
+      { text: 'Colors', link: 'colors' },
     ],
```

Comment:
please create a new section called Core and nest colors, typography and logo there

Reply:
Okay